frontend/src/scenes/actions/ActionStep.tsx

```diff
@@ -293,7 +293,7 @@ function TypeSwitcher({
                 }
             >
                 <Radio.Button value="$autocapture">Autocapture</Radio.Button>
-                <Radio.Button value="event">Event</Radio.Button>
+                <Radio.Button value="event">Custom event</Radio.Button>
```

Comment:
Think this should be Calculated event here?

Reply:
This is where the term events gets overloaded, but I reverted this back to "custom events" since it's a selector for all events not "Autocapture" and "Pageview." It shouldn't be calculated events since we don't allow actions to be created from other actions, but open to other suggestions here too!

Reply:
Got it! Yeah that makes sense now that I know the context :+1: